### PR TITLE
add apis in books genre

### PR DESCRIPTION
**Pull request by heygroot**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,9 +115,12 @@ API | Description | Auth | HTTPS | CORS |
 | [British National Bibliography](http://bnb.data.bl.uk/) | Books | No | No | Unknown |
 | [Goodreads](https://www.goodreads.com/api) | Books | `apiKey` | Yes | Unknown |
 | [Google Books](https://developers.google.com/books/) | Books | `OAuth` | Yes | Unknown |
+| [ISBN](https://isbndb.com/apidocs/v2) | Books | `apiKey` | Yes | Unknown |
 | [LibGen](http://garbage.world/posts/libgen/) | Library Genesis search engine | No | No | Unknown |
 | [Open Library](https://openlibrary.org/developers/api) | Books, book covers and related data | No | Yes | Unknown |
 | [Penguin Publishing](http://www.penguinrandomhouse.biz/webservices/rest/) | Books, book covers and related data | No | Yes | Unknown |
+| [The Bible](https://scripture.api.bible/) | Books | `apiKey` | Yes | Unknown |
```
Comment (yannbertrand) [inline README.md:122]:
```suggestion
| [The Bible](https://docs.api.bible) | Books | `apiKey` | Yes | Unknown |
```

**Review by yannbertrand — CHANGES_REQUESTED** [review]
Please update your APIs descriptions to explain what's in each one.

**Comment by amiskus999** [comment]
Hello 
Please I'm a new python programmer with skills in python data structures, please can I get projects to work on?

**Comment by matheusfelipeog** [comment]
This PR is inactive, so the API was added in another PR/commit.

More information at: #2002

**Pull request closed without merging**